Objects/typevarobject.c

```diff
@@ -23,6 +23,8 @@ typedef struct {
     PyObject *evaluate_bound;
     PyObject *constraints;
     PyObject *evaluate_constraints;
+    PyObject *default_;
```

Comment:
I don't know the exact policy for the C API, but if you were to use `default_value` in the AST layout, I think it would make sense to also call it `default_value` here and everywhere else. At least, it captures the semantic of being a default *value* and not just a default *type*.

Reply:
Sure, renaming this. Note this field is not part of the C API in any way though; it's not even exposed outside of the file.